### ISSUE TITLE
Q: ETA for 6.0?

### ISSUE THREAD

**Issue by fubar-coder** [appccelerate/statemachine#74]
It seems that a 6.0 release was planned (with nullable reference types). Is there an ETA for it?

**Comment by ursenzler** [comment]
Yes, there is a branch with what could be V6. The only change is that nullable-reference types are supported. The problem is that it introduces a lot of breaking changes to the API. So, up to now, I was not convinced that it should be released. You are the first to ask for it :-)

**Comment by fubar-coder** [comment]
@ursenzler No need to hurry, especially if you don't like the breaking changes for 6.0 😉 .
